src/components/item-adding-panel/item-adding-panel.js

```diff
@@ -64,6 +68,7 @@ const ItemAddingPanel = (props) => {
               <Field
                 name='costInGp'
                 label='Cost in gp'
+                type='tel'
```

Comment:
Стоит использовать для типа `number`. Можно совместно с этим использовать `min` и `step`, чтобы избежать отрицательных и дробных.

```suggestion
                type='number' min="1" step="1"
```

Reply:
сделала number